CMakeLists.txt

```diff
@@ -120,15 +121,32 @@ target_link_libraries(thirdparty_gflags INTERFACE
   gflags
 )
 
+set(sleigh_ENABLE_TESTS OFF)
+set(sleigh_ADDITIONAL_PATCHES "${CMAKE_CURRENT_SOURCE_DIR}/patches/sleigh/x86-ia.patch;${CMAKE_CURRENT_SOURCE_DIR}/patches/sleigh/arm-thumb.patch")
+
+# GHIDRA SLEIGH
+FetchContent_Declare(sleigh
+  GIT_REPOSITORY https://github.com/lifting-bits/sleigh.git
+  GIT_TAG master
```

Comment:
You should really use a git hash here so you don't have any phantom build errors from broken updates on `master` branch of sleigh.

I know having to manually update a git hash can be annoying, but I would really suggest using a hash to an unreleased commit until there's a release that includes your fix(es) and make an issue to keep track and then change to the release tag if/when sleigh is updated.